### PR TITLE
More tweaks to documentation for clarity

### PR DESCRIPTION
**Pull request by cabhishek**

### CODE REVIEW

**Comment by rgbkrk** [comment]
Still WIP or ready to go?

**Comment by willingc** [comment]
@cabhishek Documentation rocks :sunny: Great job.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Requires Node.js 6+ and npm 3+.
 #### Quick Start
 
 1. `git clone git@github.com:nteract/commuter.git`
```
Comment (willingc) [inline README.md]: What do you think of doing this:

1. Enter in the terminal:
```
    git clone git@github.com:nteract/commuter.git
    npm i
    npm start
```

2. Open `http://localhost:3000`

File: README.md [hunk 2]
```diff
@@ -88,21 +88,35 @@ There are few different ways to get commuter deployed on your severs:
...
+        npm run start:production
+
 1. Currently, the [Demo](https://nteract-commuter.herokuapp.com/) app is deployed on [Heroku](https://www.heroku.com/).
-Instructions for getting started and setting up heroku-cli are [here](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction).
-Once all **required env variables** are set do `git push heroku master`
-1. If you want to deploy npm published packages (**recomended**) then:
-  * Install commuter cli `npm install @nteract/commuter-cli -g`
-  * `exec commuter server` - the service is typically wrapped inside [daemontools](https://cr.yp.to/daemontools.html)
+Heroku provides [a guide for getting started with the heroku CLI](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction) which will get you
+all set up. And, once all **required env variables** are set run:
```
Comment (willingc) [inline README.md]: Perhaps:

~~And, once all~~

After all **required env variables are set**, run:

File: README.md [hunk 3]
```diff
@@ -88,21 +88,35 @@ There are few different ways to get commuter deployed on your severs:
...
+
+        git push heroku master
+
+#### From published packages (**recomended**)
```
Comment (willingc) [inline README.md]: typo:

s/recommended

**Comment by cabhishek** [comment]
@willingc Thanks for the feedback! I've updated the PR.

File: README.md (hunk 1 above)
Comment (cabhishek) [inline README.md]: `Open http://localhost:3000` needs to be part of the 1) setup instruction since it doesn't apply to Heroku (it randomly assigns port)

**Comment by rgbkrk** [comment]
🙇 Thank you @cabhishek and @willingc!

**Pull request merged**